### ISSUE TITLE
Agent crashes when consuming data from Kafka

### ISSUE THREAD

**Issue by martymartin** [robinhood/faust#346]
## Checklist

- [x] I have included information about relevant versions
- [ ] I have verified that the issue persists when using the `master` branch of Faust.

## Steps to reproduce
Consume raw data from kafka topic

## Expected behavior

Agent does not crash.

## Actual behavior

Agent crashing/throwing error. Fixed when moving from 1.6.1 -> 1.6.0.

## Full traceback

```pytb
[2019-05-09 17:13:16,197: ERROR]: [^---Agent*: hello_world.processor]: Crashed reason=TypeError('after() takes exactly 3 positional arguments (2 given)') 
Traceback (most recent call last):
  File "/anaconda3/lib/python3.7/site-packages/faust/agents/agent.py", line 601, in _execute_task
    await coro
  File "/Users/mathieurodrigue/Development/faust scripts/hello_world.py", line 14, in processor
    async for payload in stream:
  File "/anaconda3/lib/python3.7/site-packages/faust/streams.py", line 724, in _c_aiter
    it.after(event, do_ack)
  File "faust/_cython/streams.pyx", line 92, in faust._cython.streams.StreamIterator.after
TypeError: after() takes exactly 3 positional arguments (2 given)
```

# Versions

* Python version 3.7.1
* Faust version 1.6.1
* Operating system Mac OSX 10.13.4
* Kafka version 2.1.1
* RocksDB version (if applicable) NA

**Comment by dleyanlin** [comment]
have same issue in version 1.6.1

**Comment by MicicFilip** [comment]
I also have that issue that @martymartin described

**Comment by DeoLeung** [comment]
looks like is an issue with `mac`, I have a centos docker which works fine but fails on my mbp

**Comment by chroming** [comment]
I have this issue too with python 3.7.3 on mac, but with python 3.6 works fine

**Comment by ask** [comment]
This issue is only present when the Cython extensions are compiled.

It's fixed in 1.7.0!
